src/js/components/AppListStatusFilterComponent.jsx

```diff
@@ -33,18 +26,18 @@ var AppListStatusFilterComponent = React.createClass({
 
   getInitialState: function () {
     return {
-      appsStatusesCount: getInitialAppStatusCount(),
+      appsStatusesCount: {},
```

Comment:
Yes, you can do both with the state in plural.
I've read that a state that is derived from a number is preferred as "statuses" in plural.